ibm_catalog.json

```diff
@@ -306,6 +310,20 @@
                 }
               ]
             },
+            {
+              "key": "resource_group",
```

Comment:
Not following why you are adding this virtual input? You should be mapping the `existing_resource_group_name` input down

Reply:
if your starting point is buckets there is no RG, so we need a virtual for it. For consistency I will update it to `existing_resource_group_name` with displayname

Reply:
ahh got it - just for buckets yes